doc/classes/BaseMaterial3D.xml

```diff
@@ -259,8 +259,9 @@
 			For best results, the texture should be normalized (with [member heightmap_scale] reduced to compensate). In [url=https://gimp.org]GIMP[/url], this can be done using [b]Colors &gt; Auto &gt; Equalize[/b]. If the texture only uses a small part of its available range, the parallax effect may look strange, especially when the camera moves.
 			[b]Note:[/b] To reduce memory usage and improve loading times, you may be able to use a lower-resolution heightmap texture as most heightmaps are only comprised of low-frequency data.
 		</member>
-		<member name="metallic" type="float" setter="set_metallic" getter="get_metallic" default="0.0">
-			A high value makes the material appear more like a metal. Non-metals use their albedo as the diffuse color and add diffuse to the specular reflection. With non-metals, the reflection appears on top of the albedo color. Metals use their albedo as a multiplier to the specular reflection and set the diffuse color to black resulting in a tinted reflection. Materials work better when fully metal or fully non-metal, values between [code]0[/code] and [code]1[/code] should only be used for blending between metal and non-metal sections. To alter the amount of reflection use [member roughness].
+		<member name="metallic" type="float" setter="set_metallic" getter="get_metallic" default="0.0" keywords="metalness">
+			A high value makes the material appear more like a metal. Non-metals use their albedo as the diffuse color and add diffuse to the specular reflection. With non-metals, the reflection appears on top of the albedo color. Metals use their albedo as a multiplier to the specular reflection and set the diffuse color to black resulting in a tinted reflection. Materials work better when fully metal or fully non-metal, values between [code]0[/code] and [code]1[/code] should only be used for blending between metal and non-metal sections. To alter the amount of reflection, use [member roughness].
+			Values above [code]1.0[/code] can be used to further increase metalness when using a [member metallic_texture] by treating its dark spots as rougher than they would otherwise be, but the final metalness is clamped to [code]1.0[/code].
```

Comment:
```suggestion
			Values above [code]1.0[/code] can be used to further increase metalness when using a [member metallic_texture] by treating its dark spots as more metallic than they would otherwise be, but the final metalness is clamped to [code]1.0[/code].
```